checkstyle/checks.xml

```diff
@@ -50,7 +50,7 @@
 
     <module name="LineLength">
         <property name="fileExtensions" value="java"/>
-        <property name="max" value="100"/>
+        <property name="max" value="120"/>
```

Comment:
this change should be a separate PR, and it would probably be best to discuss it first in the weekly Data Platform Infrastructure Working Group meeting.

Reply:
Can do!

Reply:
reverted